docs/source/posts/2022/2023-02-01-final-report-shivam.rst

```diff
@@ -0,0 +1,406 @@
+
+-  Exporting scene data as glTF file
+-  Materials & Textures
+
+   -  Textures
+   -  PBRs
+
+-  Animations
+
+   -  Simple Actor animations
+   -  Skinning
+   -  Morphing
+
+-  Stretch Goals
+
+   -  Ability to load ``.glb`` files
+
+Objectives Completed
+--------------------
+
+Loading Static glTF models
+**************************
+
+
+   A glTF file is a JSON like file format containing required data for
+   3D scenes. VTK has two built-in glTF loaders. However, they lack
+   ability to animate and apply materials. Added methods to load binary
+   data and create actors from them. These actors can be added directly
+   to the scene. The glTF class reads texture data from either
+   ``base64`` encoded string or from the image file, and maps the
+   texture to the actor using the given UV data. It is capable of doing
+   the following:
+
+   -  Loading both ``gltf`` and ``glb`` files. Get actors from the
+      model.
+   -  Applying textures and colors from materials.
+   -  Setting cameras if the model contains multiple cameras.
+   -  Apply normals (for a smoother surface).
+
+   .. figure:: https://user-images.githubusercontent.com/74976752/174492510-b9f10816-3058-4a7b-a260-0627406354ba.png
+      :alt: image
+
+
+   *Pull Requests:*
+
+   -  **Importing glTF files: (Merged)**
+      https://github.com/fury-gl/fury/pull/600
+   -  **Loading glTF Demo (with textures) (Merged):**
+      https://github.com/fury-gl/fury/pull/600
+
+
+Exporting Scene as a glTF
+*************************
+
+
+   The fury scene can contain multiple objects such as actors, cameras,
+   textures, etc. We need to get the primitive information (such as
+   Vertices, Triangles, UVs, Normals, etc.) from these objects and store
+   them into a ``.bin`` file. Added methods that export these
+   information to a ``.gltf`` or ``.glb`` file format.
+
+   *Pull Requests:*
+
+   -  **Exporting scene as glTF: (Merged)**
+      https://github.com/fury-gl/fury/pull/630
+   -  **Exporting scene as glTF Tutorial: (Merged)**
+      https://github.com/fury-gl/fury/pull/630
+
+
+Simple Actor Animations
+***********************
+
+   Added simple actor animations (translation, rotation & scale of
+   actors) support. The animation data (transformation and timestamp) is
+   stored in buffers. It converts the binary data to ndarrays and
+   creates a timleline for each animation in glTF animations. This
+   timeline containsactors an can be added to the scene. We can animate
+   the scene by updating timline inside a timer callback.
+
+   https://user-images.githubusercontent.com/74976752/194863125-58f3717d-d89e-48e7-8e2c-a8501e4f230b.mp4
+
+   *Pull Requests:*
+
+   -  **Simple Animations in glTF: (Merged)**
+      https://github.com/fury-gl/fury/pull/643
+   -  **Simple Animations in glTF Tutorial: (Merged)**
+      https://github.com/fury-gl/fury/pull/643
+
+
+Morphing in glTF
+****************
+
+   glTF allows us to animate meshes using morph targets. A morph target
```

Comment:
you should not start with tabulations. See above. Maybe this is why videos integrations does not work: 

![image](https://user-images.githubusercontent.com/23106443/216668543-4108fe70-25e1-482f-aedf-324c4e12fb7f.png)